swagger.yaml

```diff
@@ -1518,17 +1517,20 @@ components:
         isActive:
           type: boolean
           description: Defining is user is active e.g. is employed.
+        vacationDaysByYear:
+          type: array
+          items:
+            type: string
+            description: Year and vacation days in format "YYYY:DDD".
+        unspentVacationDaysByYear:
+          type: array
+          items:
+            type: string
+            description: Year and unspent vacation days in format "YYYY:DDD".
         passedQuestionnaires:
           type: array
           items:
-            type: integer
-        severaUserId:
-          type: string
-          description: User's ID in Severa
-        forecastId:
-          type: integer
-          format: int32
-          description: User's ID in Forecast. Used for keeping a reference of Forecast based TimeEntries and Users.
+            type: string
```

Comment:
My understanding of this PR was that it was just going to remove the duplicate severaUserId property from attribute.
So I agree I'm not sure the forecastId type should be changed, but is there actually a reason for us to remove it? Maybe it's fine to remove.

As for adding in the vacationDaysByYear and unspentVacationDaysByYear have these changes been requested by Stepan related to his vacations feature?

Reply:
The ForecastID was left intact, because I am not sure yet if some part of the page will break and stop building without it - that is the only reason. 
As for all the other properties: that is how the Attributes looked in my PR, that was merged. Other merge overwritten the changes somehow and i just told Rikku to revert that to the stage it was two days ago.

Reply:
Actually, now that i check it isn`t there so I guess i had something else in mind a few days ago

Reply:
Yes, I definitely want the vacation days to be back on develop, and we can leave forecast untouched as well.

Reply:
Alright please revert forecastId to how it was before these changes and then we can approve